src/components/Task/Task.css

```diff
@@ -0,0 +1,16 @@
+.task {
```

Comment:
Just a design remark.

This component should not occupy all the width of the screen. It should be something like 600px in width
![Captura de pantalla 2021-11-08 a las 13 12 27](https://user-images.githubusercontent.com/15636998/140740080-9a07eeac-35b6-4dd2-9362-ee1aad61d318.png)
h: